tools/accuracy_checker/configs/pelee_coco.yml

```diff
@@ -0,0 +1,34 @@
+models:
+  - name: peleenet_coco
+
+    launchers:
+      - framework: dlsdk
+        tags:
+          - FP32
+        model:   peleenet_coco/FP32/peleenet_coco.xml
```

Comment:
Wrong path. Ought to be `public/pelee_coco/FP32/pelee_coco.*` (in fact, `public/pelee-coco/FP32/pelee-coco.*` :) )